client-js/app/elements/labrad-plot.ts

```diff
@@ -177,12 +177,13 @@ export class Plot extends polymer.Base {
             .on('zoom', () => this.handleZoom_());
 
     // Plot area.
+    const marginLeft = p.margin.left;
+    const marginTop = p.margin.top;
     p.svg = d3.select(area)
-            .append('svg:svg')
-            .attr('width', width + p.margin.left + p.margin.right)
-            .attr('height', height + p. margin.top + p.margin.bottom)
-            .append('g')
-            .attr('transform', `translate(${p.margin.left}, ${p.margin.top})`);
+              .append('svg:svg')
+                .attr('class', 'flex')
```

Comment:
The indenting change might make this hard to see, but I removed width and height here on purpose so that the SVG can resize with its parent container.